content/installation/java/standard/ContrastUI.md

```diff
@@ -1,24 +1,22 @@
 <!--
-title: "Java Agent Installation"
-description: "Overview of the Java Agent installation"
-tags: "installation Java agent overview installation"
+title: "Download the Java Agent from the Contrast UI"
+description: "Downloading the Java agent from the Contrast UI"
+tags: "installation Java agent installation contrast ui download"
 -->
 
-To install the Java agent, complete the following steps:
+To install the Java agent from the Contrast UI, complete the following steps:
 
 * Log in to the Contrast UI.
 * Click the button in the top navigation bar to **Add Agent**.
 * Select the **Java** agent in the dropdown menu, and click the button to **Download Agent**. You may need to specify proxy authentication information required by your network before downloading the agent.
```

Comment:
This step will change slightly after we update the wizard. The first step will ask them if they want to do Host Installation or Standard Installation. On the Standard Installation path, they will need to check a box to choose to download directly from teamserver -- the default instructions will be for downloading from system package managers.

Reply:
Just FYI - nothing to change for this PR.